README.adoc

```diff
@@ -88,9 +91,13 @@ You can then activate the profile using `-Pspanner` such as `mvn compile -Pspann
 
 == User Guide
 
-The following section describes recommendations and best practices for how to best use the Cloud Spanner Hibernate Dialect.
+This guide contains a variety of best practices for using Hibernate with Spanner which can significantly improve the performance of your application.
+
+=== Schema Creation and Entity Design
 
-=== Use Generated UUIDs for ID Generation
+Hibernate generates statements based on your Hibernate entity's design. Following these practices can result in better DDL and DML statement generation which can improve performance.
```

Comment:
```suggestion
Hibernate generates statements based on your Hibernate entity design. Following these practices can result in better DDL and DML statement generation which can improve performance.
```

Reply:
Because it's going to be based on multiple entities, and doing "entity's" vs "entities" is too pedantic for me. "entity design" covers both.

Reply:
Done.